langchain/llms/huggingface_hub.py

```diff
@@ -69,14 +69,22 @@ def validate_environment(cls, values: Dict) -> Dict:
     @property
     def _default_params(self) -> Mapping[str, Any]:
         """Get the default parameters for calling HuggingFace Hub API."""
+        # Convert temperature from [0, 1] to [1, 100] so 0 maps to 1 and 1 maps to 100.
```

Comment:
Huggingface seems to have weird expectation that temperature is on a [1, 100] scale rather than [0, 1]